lib/ui/views/splash/widgets/curved_clipper.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class CurvedShapeClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    final path = Path();
+
+    print('device height ${size.height}');
```

Comment:
this would show in production, kindly make use of the logger class as this is now available for use or u delete it if not in use.

Reply:
Fixed using the Logger class